packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -1633,6 +1633,34 @@ const lroPatch202 = (patchOp, _opts, ctx) => {
     return errors;
 };
 
+const lroPostReturn = (postOp, _opts, ctx) => {
+    if (postOp === null || typeof postOp !== "object") {
+        return [];
+    }
+    const path = ctx.path || [];
+    const errors = [];
+    const responses = postOp.responses;
+    if (responses && (!responses["200"] || !responses["202"])) {
```

Comment:
>200 [](http://example.com/codeflow?start=34&length=3)

lets only flag 202 as the "required" response code for LRO post. The 200 is no longer required.

Reply:
This was part of #450, not part of this PR. Made an ADO item to address this.